.github/workflows/clair-in-ci.db.yaml

```diff
@@ -46,21 +46,21 @@ jobs:
       - name: Pull-and-retag-image
         if: ${{ github.event.client_payload.pr_event != 'pull_request' }}  # don't retag image from PR
         run: |
-          docker pull ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
-          docker tag ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.PREVIOUS_TAG }}
-          docker push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.PREVIOUS_TAG }}
-          docker rmi -f $(docker images -q)
+          podman pull ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }}
+          podman tag ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.LATEST_TAG }} ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.PREVIOUS_TAG }}
+          podman push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.PREVIOUS_TAG }}
+          podman rmi -f $(podman images -q)
```

Comment:
https://github.com/redhat-appstudio/clair-in-ci-db/pull/6 I wrote PR to use tags in the exactly the same way as ClamAV, it's just failing I don't know why

Reply:
Do you mean that we should get rid of the `previous` tag in order to align with ClamAV?  Doesn't keeping the tag give us some redundancy in case we need to rollback?

Reply:
calculated timestamp is unique tag, that won't change, we will use it for rollbacks if needed.
Tags latest and v1 are floating and updated after testing, and we should push to both, latest for bw compatibility.

Reply:
Ack, I'll wait for #6 to be merged then update the tagging here accordingly